cmd/clairctl/clair/health.go

```diff
@@ -0,0 +1,26 @@
+package clair
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+)
+
+func IsHealthy() bool {
+	healthURI := strings.Replace(uri, "6060/v1", strconv.Itoa(healthPort), 1) + "/health"
```

Comment:
That's sketchy. What happens if Clair isn't listening on 6060. Should get the Clair's endpoints (API without `/v1/`, and health) using config. Then internally because clairctl relies on the v1 API, `v1/` should be appended in the calls. Proper defaults should do the trick.